redisinsight/ui/src/components/database-overview/DatabaseOverview.tsx

```diff
@@ -56,46 +63,69 @@ const DatabaseOverview = (props: Props) => {
-                  <EuiFlexItem
-                    className={cx(styles.overviewItem, overviewItem.className ?? '')}
-                    key={overviewItem.id}
-                    data-test-subj={overviewItem.id}
-                    grow={false}
+            {
+              metrics?.map((overviewItem) => (
+                <EuiFlexItem
+                  className={cx(styles.overviewItem, overviewItem.className ?? '')}
+                  key={overviewItem.id}
+                  data-test-subj={overviewItem.id}
+                  grow={false}
+                >
+                  <EuiToolTip
+                    position="bottom"
+                    className={styles.tooltip}
+                    content={getTooltipContent(overviewItem)}
                   >
-                    <EuiToolTip
-                      position="bottom"
-                      className={styles.tooltip}
-                      content={getTooltipContent(overviewItem)}
-                    >
-                      <EuiFlexGroup gutterSize="none" responsive={false} alignItems="center" justifyContent="center">
-                        {overviewItem.icon && (
-                          <EuiFlexItem grow={false}>
-                            <EuiIcon
-                              size="m"
-                              type={overviewItem.icon}
-                              className={styles.icon}
-                            />
-                          </EuiFlexItem>
-                        )}
-                        <EuiFlexItem grow={false} className={styles.overviewItemContent}>
-                          {overviewItem.content}
+                    <EuiFlexGroup gutterSize="none" responsive={false} alignItems="center" justifyContent="center">
+                      {overviewItem.icon && (
+                        <EuiFlexItem grow={false} className={styles.icon}>
+                          <EuiIcon
+                            size="m"
+                            type={overviewItem.icon}
+                            className={styles.icon}
+                          />
                         </EuiFlexItem>
-                      </EuiFlexGroup>
-                    </EuiToolTip>
-                  </EuiFlexItem>
-                ))
-              }
-            </EuiFlexGroup>
-          </div>
+                      )}
+                      <EuiFlexItem grow={false} className={styles.overviewItemContent}>
+                        {overviewItem.content}
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiToolTip>
+                </EuiFlexItem>
+              ))
+            }
+            <EuiFlexItem
+              className={cx(styles.overviewItem, styles.autoRefresh)}
+              grow={false}
+              data-testid="overview-auto-refresh"
+            >
+              <EuiFlexItem grow={false} className={styles.overviewItemContent}>
+                <AutoRefresh
```

Comment:
AutoRefresh is turned off by default. Is it right behaviour? I think the auto-activation should stay

Reply:
Updated